spec/controllers/idv/hybrid_handoff_controller_spec.rb

```diff
@@ -382,10 +382,32 @@
         }
       end
 
-      it 'sends analytics_submitted event for desktop' do
-        put :update, params: params
+      context 'vendor is socure' do
+        before do
+          subject.document_capture_session.update(doc_auth_vendor: Idp::Constants::Vendors::SOCURE)
+        end
 
-        expect(@analytics).to have_logged_event(analytics_name, analytics_args)
+        it 'sends analytics_submitted event for desktop' do
+          put :update, params: params
+
+          expect(@analytics).to have_logged_event(analytics_name, analytics_args)
+          expect(subject.document_capture_session.doc_auth_vendor)
+            .to eq(Idp::Constants::Vendors::SOCURE)
+        end
+
+        context 'when selfie_deskto_test_mode is enabled' do
+          let(:desktop_test_mode_enabled) { true }
```

Comment:
Given we're tesing the wrong config var, are we sure this spec is testing what we want?

```suggestion
          let(:desktop_selfie_test_mode_enabled) { true }
```

Reply:
Yup!